x-pack/plugins/lists/server/saved_objects/exception_list.ts

```diff
@@ -46,7 +46,12 @@ export const commonMapping: SavedObjectsType['mappings'] = {
       type: 'keyword',
     },
     name: {
-      type: 'keyword',
+      fields: {
```

Comment:
FYI: Left the property `type` as is because exception list type is either `detections` or `endpoint` and I believe we use the standard ES tokenizer which tokenizes based on word boundaries.